emm-docs/content/getting-started/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+weight: 1
```

Comment:
Note: `weight` lets you configure the order in which the pages will appears in the navigation.